backend.php

```diff
@@ -81,7 +81,38 @@
 
         echo '<img src="data:image/png;base64, '.$based.'" alt="nice" style="margin-bottom: 10px;"/>';
         break;
-    
+     case 'nft':
+        if ($_SERVER['REQUEST_METHOD'] !== 'GET' || empty($_GET['html'])) {
```

Comment:
You appeared to copy `empty($_GET['html'])` from the request above this case. I do not believe you have a need for that check at this time, especially dealing with critical blonkChain transactions, we want to avoid as many CPU cycles as possible! Otherwise, LGTM ✅